mods/ra2/rules/defaults.yaml

```diff
@@ -392,6 +392,7 @@
 		Voice: Attack
 	ActorLostNotification:
 	BodyOrientation:
+		QuantizedFacings: 32
```

Comment:
OK, changing it then, but should i change ones on `^Vehicle` and `^Ship` on this PR or open a new PR for them.